src/app/main.cpp

```diff
@@ -195,7 +197,6 @@ int main( int argc, char **argv )
 #endif
 #endif
 
-  qInstallMessageHandler( qfMessageHandler );
```

Comment:
What was wrong here?

Reply:
I just realized you're using this for sentry too. What happens here is that on linux builds, we're neither ANDROID nor SENTRY, and yet we register the message handler, which as a result eats all messages and outputs nothing.

Reply:
@m-kuhn , pushed better fix.